staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external_test.go

```diff
@@ -1262,3 +1333,108 @@ func TestNeedToUpdateHttpHealthChecks(t *testing.T) {
 		})
 	}
 }
+
+func TestFirewallObject(t *testing.T) {
+	t.Parallel()
+	vals := DefaultTestClusterValues()
+	gce, err := fakeGCECloud(vals)
+	gce.nodeTags = []string{"node-tags"}
+	require.NoError(t, err)
+	srcRanges := []string{"10.10.0.0/24", "10.20.0.0/24"}
+	sourceRanges, _ := utilnet.ParseIPNets(srcRanges...)
```

Comment:
@freehan @cheftako -- `k8s.io/utils/net.ParseIPNets` returns an `IPNetSet` (which is like a map) and the `StringSlice` method doesn't deterministically return a list of strings. The test is failing for that reason.

Can you please fix this? Or, should this PR be reverted, for now, considering that Code Freeze is very near and presubmits are failing?

Ref: 
1. https://prow.k8s.io/job-history/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-unit
2. Some failed Jobs:
  - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/99641/pull-kubernetes-unit/1367778084027109376
  - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/99423/pull-kubernetes-unit/1367803543393669120
  - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/99817/pull-kubernetes-unit/1367776489864433664
  - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/98883/pull-kubernetes-unit/1367785835595829248
  - https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/99210/pull-kubernetes-unit/1367783644051017728

xref: https://kubernetes.slack.com/archives/C2C40FMNF/p1614942354205800

Reply:
PR revert by @aojea in https://github.com/kubernetes/kubernetes/pull/99837

Reply:
Thank you folks! Fix is in #99858